stripe/res/layout/activity_payment_sheet.xml

```diff
@@ -48,7 +48,7 @@
                 android:layout_height="wrap_content"
                 android:visibility="gone"
                 android:layout_marginVertical="2dp"
-                android:layout_marginHorizontal="10dp"
+                android:layout_marginHorizontal="23dp"
                 style="@style/StripePaymentSheetErrorMessage" />
         </LinearLayout>
```

Comment:
Is the error text color on the form purposefully different than the color of the error message?     
   ``` <color name="stripe_paymentsheet_error_text">#DC2727</color>
    <color name="stripe_paymentsheet_form_error">#CF6679</color>```